packages/gestalt-design-tokens/tokens/classic/comp-web-color-light.json

```diff
@@ -380,6 +380,22 @@
           "value": "#111"
         }
       },
+      "iconbutton": {
```

Comment:
this needs it counterpart in the dark file.

Reply:
Fixed